### PR TITLE
Update design-tokens.yaml

### PR DESCRIPTION
**Pull request by Blackbaud-AdamFunderburk**
This update adds 7 new colors for categorized containers in SKY UX.

### CODE REVIEW

File: src/design-tokens.yaml [hunk 1]
```diff
@@ -79,7 +87,16 @@ background-color:
   danger: *color-red-30
   danger-dark: *color-red-60
   page-default: *color-white
-
+  
+container-color:
```
Comment (Blackbaud-SteveBrush) [inline src/design-tokens.yaml]: The section above is called "background-color". Are these also background colors?

Comment (Blackbaud-SteveBrush) [reply src/design-tokens.yaml]: I found a docs PR that answers my question, above. However, looks like John L. had a question about the name, too: https://github.com/blackbaud/skyux2-docs/pull/610#pullrequestreview-275210943

Comment (Blackbaud-AdamFunderburk) [reply src/design-tokens.yaml]: Todd and I discussed the token scheme yesterday and I think we've got a better direction for naming and organizing these. Stay tuned.

**Comment by Blackbaud-ToddRoberts** [comment]
@Blackbaud-SteveBrush are there any updates we need to make to the token processor with the new structure of the color tokens we are adding (with the additional level)?

**Pull request merged**